puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -60,7 +60,10 @@
           values:
             - fast
     execution-strategy:
-      combination-filter: "!(foreman_branch == 'develop' && ruby == '1.9.3')"
+      combination-filter: >
+        "(foreman_branch != 'develop' && ruby == '1.9.3' && database == 'postgresql') ||
+        (database == 'postgresql') ||
```

Comment:
This will run the Ruby 1.9.3 on PostgreSQL test on develop, but Ruby 1.9.3 should be excluded.